src/dune_rules/dune_file.ml

```diff
@@ -45,6 +45,117 @@ module Js_of_ocaml = struct
     { flags = Ordered_set_lang.Unexpanded.standard; javascript_files = [] }
 end
 
+module Ctypes = struct
+  module Build_flags_resolver = struct
+
+    module Vendored = struct
+      type t =
+        { c_flags : Ordered_set_lang.Unexpanded.t
+        ; c_library_flags : Ordered_set_lang.Unexpanded.t }
+
+      let decode =
+        fields
+          (let+ c_flags = Ordered_set_lang.Unexpanded.field "c_flags"
+           and+ c_library_flags = Ordered_set_lang.Unexpanded.field "c_library_flags" in
+           { c_flags; c_library_flags })
+    end
+
+    type t =
+      | Pkg_config
+      | Vendored of Vendored.t
+
+    let decode =
+      let vendored =
+        let+ p = Vendored.decode in
+        Vendored p
+      in
+      sum [ ("pkg_config" , return Pkg_config)
+          ; ("vendored"   , vendored ) ]
+
+    let default = Pkg_config
+  end
+
+  module Concurrency_policy = struct
+    type t =
+      | Sequential
+      | Unlocked
+      | Lwt_jobs
+      | Lwt_preemptive
```

Comment:
I'm unfamiliar with the lwt jobs. Where do these come from?

Reply:
I haven't used them either, though I notice they're defined here: https://ocamllabs.io/ocaml-ctypes/Cstubs.html#TYPEconcurrency_policy